docs/dev/table/hive/index.md

```diff
@@ -79,7 +79,7 @@ To integrate with Hive, users need the following dependencies in their project.
 <dependency>
   <groupId>org.apache.flink</groupId>
   <artifactId>flink-shaded-hadoop-2-uber</artifactId>
-  <version>2.7.5-{{site.version}}</version>
+  <version>2.7.5-7.0</version>
```

Comment:
As we discussed in #9532 , we should add a `shaded-version` in `docs/_config.yml`, and use the variables in the documentation instead of hard code.

Reply:
Hi, @wuchong , Thank you ,I will add a `shaded-version`  in  `docs/_config.yml`.

Reply:
And ,how can we add comments to be easy to understand? @wuchong @bowenli86

Reply:
I want to add comment in `docs/_config.yml` as follows 
`# Plain flink-shaded version is needed for e.g. the hive connector.
`
`shaded_version: "7.0"`
Do you think that is ok? Thank you! @wuchong @bowenli86 @zentol

Reply:
Yes. That looks good to me. Please add a note after that. 

"Please update the version once new flink-shaded is released."

Reply:
Thanks ,I got it, and I had added this sentence  after that.
`# Plain flink-shaded version is needed for e.g. the hive connector.
`
`# Please update the shaded_version once new flink-shaded is released.
`
`
shaded_version: "7.0"
`